### PR TITLE
Home page content

### PR DESCRIPTION
**Pull request by order4adwriter**
hello @camelaissani 
here is the completed content for your application's website home page . please take a look, review it and merge it to the master repository. 

[Loca homepage.docx](https://github.com/camelaissani/loca/files/1845573/Loca.homepage.docx)
completed copy for the home page.

Thanks.

**PS:** working on the about us page content now...

### CODE REVIEW

**Comment by camelaissani** [comment]
Awesome! I guess you want your text on the web home page right?

**Comment by order4adwriter** [comment]
oh  that'd be nice ...if you can do that ...thanks

**Comment by order4adwriter** [comment]
have you merged the files?

**Comment by order4adwriter** [comment]
wow....still no answer  from you?

That is unfortunate...

**Comment by camelaissani** [comment]
Okay okay I'm gonna review your contribution today...

**Comment by order4adwriter** [comment]
okay thanks, looking forward to your review.... please merge the files when you  do so and it all looks good.

Thanks.

File: About Loca [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+                                                About Loca
+
+Why pay hundreds of dollars for an overpriced, problematic and confusing rental property management software, when you can create a similar app to your taste and preferences, without any  of the complex or complicated features… and best of all, absolutely free to use anytime, anywhere?
+
+This is the underlying philosophy behind the creation and development of Loca by its Lead Developer, Camel Aissani. A rental property manager by profession who also enjoys Software Development and Coding as a part time hobby.
+
+From the initial plan of creating Loca to help himself and his fellow rental property manager cum friend in their businesses, Loca has evolved into  a fully integrated web-based, interactive and no-cost solution.
+
+Loca is currently helping thousands of rental estate professionals globally to effectively manage their businesses, keep accurate and secure rental records, significantly reduce the need for making frequent, costly phone calls to tenants or vendors and avoid most of the tiresome paperwork associated with managing a rental property.
```
Comment (camelaissani) [inline About Loca]: Unfortunately, I cannot confirm this point. I don't have any metrics to count the number of people using Loca.

Comment (camelaissani) [inline About Loca]: Unfortunately, I cannot confirm this point. I don't have any metrics to count the number of people using Loca.

File: Home Page Content [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+For Property Managers, By Property Managers …
+Welcome To The World Of Loca
+
+
+
+
+
+An Advanced Web-based, Open Source Management Application Created Specifically for Rental Property Owners, Landlords and Managers
+  
+Available for Windows, Mac and Linux Users.
```
Comment (camelaissani) [inline Home Page Content]: As it is a web-based app, I think it's useless to mention those Operating Systems.

File: Home Page Content [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Try out Our Demo Here
+
+Loca is a simple but professional, user-friendly and eye-catching Rental Property Management Application.
+
+Professional Property Management, The Stress-Free Way
+It is designed to give you the crucial advantage, customizable solutions and necessary accounting intelligence to help you make sound rental property management decisions and control your rental property expenses effectively.
+
+Best Of All, It’s Totally Free To Use… No Hidden Charges, No cutthroat upfront fees, Nothing!
```
Comment (camelaissani) [inline Home Page Content]: So far, I haven't deployed Loca for business purposes. I don't commercialize this app.
What I understand of your lines above, is that I offer a ready to use solution for professional property managers.
Unfortunately, it is not the case so far. You have to clone my code, do some customizations here and there, and run it on a server.

File: Home Page Content [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Growing Your Rental Businesses Has Never Been This Easy…
+Whether it’s a single property or a large property portfolio, Loca helps you get it done.
+
+Register Here OR Try out Our Demo Here
```
Comment (camelaissani) [inline Home Page Content]: User cannot register so far

File: Home Page Content [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+So What Make Loca So Special Anyway?
+•	Secure Online Cloud based Solution.
+•	No need for any software download, backup or update.
+•	Smart, user-friendly and responsive layout. No need to waste too much time learning the system.
+•	Available for use by all mobile/ web devices & accessible from anywhere in the world with an internet connection.
```
Comment (camelaissani) [inline Home Page Content]: responsive layout and mobile is not supported

File: Home Page Content [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+•	It is open-source… therefore it is totally free to use for all properties. No upfront costs.
+
+•	Created by experienced rental property managers, so your rental property management needs are in good hands.
+
+Register Here OR Try Out Our Demo Here
```
Comment (camelaissani) [inline Home Page Content]: Same as line 26

**Comment by camelaissani** [comment]
Thanks a lot for this contribution, I really like what you've done. However, IMO it's too business oriented for an open source project.
Could you make some changes and adapt your text?
Also, could you use the Markdown syntax? 
What do you think to help me to rewrite the [README.md](https://github.com/camelaissani/loca/blob/master/README.md) and merge your work in this page?

**Comment by order4adwriter** [comment]
Hi,

Thanks for your review. From your comments it seems you'd like me to tone down the language of both page content... make it a little less formal/ professional sounding and more appealing to casual landlords looking for something simple but efficient to use.

Alright, will do some editing on the contents later today and send to you for your review . Here's hoping u can review, approve and merge them today or as soon as possible.

About writing the README.md ...you just the words right out of my mouth...lol!
I was gonna ask you about writing it and sure, I can start writing it  as soon as I'm done with the editing.

Thanks again for your interest.

**Comment by order4adwriter** [comment]
Hello,

here are the updated versions of the content i sent earlier
[About Loca rehashed.docx](https://github.com/camelaissani/loca/files/1859938/About.Loca.rehashed.docx)
[Loca homepage edited.docx](https://github.com/camelaissani/loca/files/1859939/Loca.homepage.edited.docx)

**Comment by order4adwriter** [comment]
#### About Loca

Why pay hundreds of dollars for an overpriced, problematic and confusing rental property management software, when you can create a similar and simple app to your taste and preferences, without any  of the complex or complicated features… and best of all, absolutely free to use anytime, anywhere?

This is the underlying philosophy behind the creation and development of Loca by its Lead Developer, Camel Aissani. A rental property manager by profession who also enjoys Software Development and Coding as a part time hobby.

From the initial plan of creating Loca to help himself and his fellow rental property manager cum friend in their businesses, Loca has evolved into a user-friendly, durable and no-cost solution for rental property management.

Loca is ideal for small, independent landlords and property managers to effectively manage their businesses, keep accurate and secure rental records, significantly reduce the need for making frequent, costly phone calls to tenants and vendors and avoid most of the tiresome paperwork associated with managing a rental property.

Our promise to you our esteemed clients is that Loca will be a welcome solution to your rental management needs, not an additional problem.

Thank You For Choosing Loca Today.

**Comment by order4adwriter** [comment]
For Property Managers, By Property Managers …

### Welcome To The World Of Loca


An Simple Web-based, Open Source Management Application Created Specifically for Rental Property Owners, Landlords and Managers
  

[Try out Our Demo Here
](https://demo.nuageprive.fr/dashboard#)
Loca is a simple but professional, user-friendly and eye-catching Rental Property Management Application.

### Rental Property Management, The Stress-Free Way
It is designed to help you make sound rental property management decisions and control your rental property expenses effectively.

Best Of All, It’s Totally Free To Use… No Hidden Charges, No Cutthroat upfront fees, Nothing!

### Growing Your Rental Businesses Has Never Been This Easy…
Whether it’s a single property or a multiple properties, Loca helps you get it done.

[Try out Our Demo Here
](https://demo.nuageprive.fr/dashboard#)

### So What Make Loca So Special Anyway?
•	Secure Online Cloud based Solution.
•	No need for any software download, backup or update.
•	Smart, user-friendly layout. No need to waste too much time learning the system.
•	It is open-source… therefore it is totally free to use for all properties. No upfront costs.
•	Created by experienced rental property managers, so your rental property management needs are in good hands.

[Try Out Our Demo Here](https://demo.nuageprive.fr/dashboard#)
### Switch Over To Loca Now And Enjoy The Endless Loca Advantages…

### Rent
•	Track rent payments as they come in.
•	Collect rent payments online.
•	Set up late payment remainder emails for defaulting tenants.
•	Get notified immediately the rents get paid.
•	Get a detailed tenant timeline and payment history.
•	So much more…



### Documentation
•	Create beautiful, professional rental invoices or lease contracts in a few minutes.
•	Keep accurate records of your tenants repair requests or complaints’
•	Notify them immediately by email or sms when the issue is resolved.
•	Store all crucial rental documents, tenancy info and other important files in our cloud-based digital archives and view or retrieve them via downloads at any time.
•	So much more…


### Accounting
•	Easily setup, read and update accounting reports, especially for taxation purposes.
•	Create monitoring reports for businesses alongside maintenance expenses.
•	Reduce the stressful need for endless and costly phone calls or paperwork.
•	So much more…

Starting Now, Be In Absolute Control Of Your Rental Property Cashflow… No Matter Where You Are.
[Try Out Our Demo First.](https://demo.nuageprive.fr/dashboard#)

**Comment by order4adwriter** [comment]
Okay...all done. Looking forward to your feedback.
Please merge the files to the master when you're done.

Thanks.

**Comment by camelaissani** [comment]
Should we close this PR?

**Comment by order4adwriter** [comment]
Yeah...if all 3 content have been merged.....

**Pull request closed without merging**